test/specs/calculator.add.spec.js

```diff
@@ -0,0 +1,158 @@
+const Calculator = require('../../app/calculator');
```

Comment:
Why did you choose the `calculator.add.spec.js` file name for the file if you actually check the `multiply()` method too?